t/pipe.t

```diff
@@ -0,0 +1,1439 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use Test::Nginx::Socket::Lua;
+use Cwd qw(abs_path realpath cwd);
+use File::Basename;
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 3 + 16);
```

Comment:
On my local setup, only 276 tests are executed (vs 278 on Travis CI and according to this test plan). Yet I am using the master branch of Test::Nginx.

Reply:
It is strange...
In my local machine, 278 tests are run, like one in the Travis CI.

Reply:
Resolved.